modules/ocf_ldap/files/sasl2-slapd

```diff
@@ -1,2 +1,2 @@
-mech_list:      GSSAPI
+mech_list:      GSSAPI plain
```

Comment:
I'm still not convinced this is the correct way to do this, since I feel like passwords should get passed through using SASL to Kerberos without this if they're set up correctly, and having plaintext passwords seems kinda scary.

Reply:
I don't believe this actually stores passwords in plaintext, or in a second place at all. The "plaintext" password is getting passed to saslauthd, which is configured to send it to Kerberos. I don't think this is any more insecure than sending passwords to death over https, which is what the rest of our services do.

Reply:
Just did some more investigation and you are actually right-- this line is not needed. I will delete it.